components/esp_wifi_remote/idf_component.yml

```
@@ -6,5 +6,5 @@ dependencies:
     version: '>=0.1'
   idf:
     version: '>=5.3'
-#  espressif/esp_hosted:
-#    version: '*'
+  espressif/esp_hosted:
+    version: '*'
```

I'd suggest using a little more specific version, e.g.
```suggestion
    version: '>=0.0.5'
```